pages/previewPage.html

```diff
@@ -252,6 +253,22 @@
             break;
         }
       });
+
+      window.addEventListener(
+        "click",
+        (event) => {
+          var domNode = event.target;
+          while (domNode !== null) {
+            console.log(domNode.tagName);
+            if (domNode.tagName.toLowerCase() === "a" && domNode.href !== null) {
+              vscode.postMessage({ command: "clickOnTag_A", link: domNode.href.baseVal });
+              return;
+            }
+            domNode = domNode.parentNode;
+          }
```

Comment:
I think you can use this for simplicity:

`const link = event.target.closest("a[href]")`

Reply:
I learn something new every day.